lib/trino-record-decoder/src/main/java/io/trino/decoder/DecoderColumnHandle.java

```diff
@@ -33,4 +36,9 @@
     String getMapping();
 
     String getDataFormat();
+
+    default Optional<List<String>> getOneOfs()
```

Comment:
nit: Let's consitently use `oneof` instead of `oneOf` since that's what the protobuf schema calls it.